polaris.shopify.com/content/patterns/new-features/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: New Features
+description: Highlight new features for merchants
+lede: Help merchants discover new and impactful features.
+url: /patterns/new-features
+previewImg: /images/patterns/new-feature/new-feature-cover.png
+githubDiscussionsLink: https://github.com/Shopify/polaris/discussions/6751
+variants:
+  - 'variants/default.md'
+---
+
+<div as="Variants"></div>
+
+<div as="Stack" gap="4">
+
+## Related resources
+
+- See another two-column layout in use in the [Resource detail layout](/patterns/resource-details-layout) pattern.
+- See a single-column layout in use in the [Resource index layout](/patterns/resource-index-layout) pattern.
+- Learn more about [Layout](https://shopify.dev/apps/design-guidelines/layout) in the app design guidelines.
+- Check out the Polaris [Space](/design/space) guidelines to understand Polaris grid and spacing scale.
```

Comment:
Are layouts or space is related to new feature? or is this copied over from somewhere else?

Reply:
Yes. My bad. Will remove